code-env/python/desc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "acceptedPythonInterpreters": ["PYTHON37"],
```

Comment:
Is this their requirement or ours. 3.7 is pretty old.

Reply:
It is there requirement. Although they did put out some updates, so would be good to check if they allow 3.8 or 3.9 now.

Reply:
I did some investigation and it could be possible to use newer versions of python, however that would require installing those versions on the operating system that is running dataiku dss. With the core installation, only 3.7 is provided. Due to that limitation I think we should stick with 3.7 for now.